libs/ledger-live-common/src/data/icons/svg/BCERT.svg

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 360 360">
+  <path d="M183,9.87C91.47,9.87,17,84.34,17,175.87s74.47,166,166,166,166-74.47,166-166S274.53,9.87,183,9.87Zm.12,318.39c-84.5,0-153.25-68.74-153.25-153.24S98.62,21.77,183.12,21.77s153.23,68.75,153.23,153.25-68.74,153.24-153.23,153.24Z"/>
+  <g>
```

Comment:
you can remove all the group tags `g` as they are unused and non necessary here.